fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -670,8 +678,9 @@ struct LogGenerationData : NonCopyable, public ReferenceCounted<LogGenerationDat
 	}
 
 	// For a given version, get the serialized messages
-	Optional<std::pair<Version, StringRef>> getSerializedTLogData(const Version& version,
-	                                                              const StorageTeamID& strorageTeamID);
+	Optional<std::pair<Version, std::pair<StringRef, Arena>>> getSerializedTLogData(
```

Comment:
Nit: replace `std::pair<StringRef, Arena>` with `Standalone<StringRef>`. This can be done in another PR.

Reply:
will do in a separate PR

Reply:
https://github.com/apple/foundationdb/pull/6448 please check the follow-up PR when you have time, thanks!